src/assets/js/pages/UsersPage/reducers/Columns.js

```diff
@@ -15,7 +15,7 @@ const initialState = [
   },
   {
     Header: 'Email',
-    accessor: 'email'
+    accessor: 'event.email'
```

Comment:
This column refers to the columns that we can sort users by, not the event email.